.travis/molecule.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# Avoid lint error
+sed -i "s/{%/# {%/g" roles/postgres/templates/postgres.deployment.yaml
```

Comment:
What is the error? 

Also, the change seems to have been introduced by this commit:
17f44c0227a11615de3bdaa3ab793da6d5017190
(I looked into this way too much, I initially thought this was an ansible playbook rather than a templated kubernetes yml file.)

Reply:
```
./roles/postgres/templates/postgres.deployment.yaml
  28:2      error    syntax error: found character '%' that cannot start any token (syntax)
```
https://travis-ci.com/github/fao89/pulp-operator/jobs/357436578

Reply:
Let's rename our kubernetes yaml it to .j2. Here's an example of it being named that way:
https://github.com/geerlingguy/tower-operator/blob/master/roles/tower/tasks/main.yml#L5